### PR TITLE
add test for GetPVCNameFromVolumeMountName

### PR DESCRIPTION
**Pull request by syamgk**
add test for GetPVCNameFromVolumeMountName

### CODE REVIEW

**Comment by cdrage** [comment]
LGTM

File: pkg/occlient/occlient_test.go [hunk 1]
```diff
@@ -15,6 +15,110 @@ import (
...
+								},
+							},
+						},
+					},
+				},
+			},
+			want: "test-pvc",
+		},
+		{
+			name: "Test case : 2",
```
Comment (ashetty1) [inline pkg/occlient/occlient_test.go]: Would be good to have a descriptive test name, eg. `Test case 2: Non-existent PVC`

Comment (syamgk) [reply pkg/occlient/occlient_test.go]: added names for test cases, ready for review now

**Comment by cdrage** [comment]
Changes to the name LGTM.

**Comment by kadel** [comment]
@ashetty1 can you please check this PR and if your comments were addressed please merge it.

**Comment by cdrage** [comment]
Needs rebase, after rebase + @ashetty1 's approval, let's merge.

**Pull request merged**